reboot-core/src/test/kotlin/nl/thanus/reboot/ReBootKtTest.kt

```diff
@@ -0,0 +1,34 @@
+package nl.thanus.reboot
+
+import com.github.ajalt.clikt.core.BadParameterValue
+import com.github.ajalt.clikt.core.MissingParameter
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertDoesNotThrow
+import org.junit.jupiter.api.assertThrows
```

Comment:
The project is using AssertJ for assertions. Do you mind to change these assertions to AssertJ?